curriculum/challenges/english/10-coding-interview-prep/rosetta-code/i-before-e-except-after-c.md

```diff
@@ -27,6 +27,8 @@ If both sub-phrases are plausible then the original phrase can be said to be pla
 
 Write a function that accepts a word and check if the word follows this rule. The function should return true if the word follows the rule and false if it does not.
 
+here are the words tested: receive, science, imperceivable, inconceivable, insufficient, and omniscient
```

Comment:
This just repeats the information from the test suite. I don't think this is needed.

Reply:
You don't need to visit an external site to read the list of tests that we're running?

Reply:
That is actually true🤣